File: integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/validation/VirksertCertificateException.java
package no.difi.meldingsutveksling.validation;

public class VirksertCertificateException extends RuntimeException {
    public VirksertCertificateException(Throwable t) {
        super(t);
    }
    public VirksertCertificateException(String s) {
        super(s);
    }
    public VirksertCertificateException(String s, Throwable t) {
        super(s, t);
    }
}

File: integrasjonspunkt/src/main/kotlin/no/difi/meldingsutveksling/validation/VirksertCertificateException.kt
package no.difi.meldingsutveksling.validation

class VirksertCertificateException : RuntimeException {
    constructor(t: Throwable) : super(t)
    constructor(s: String) : super(s)
    constructor(s: String, t: Throwable) : super(s, t)
}